internal/resolver/resolver.go

```diff
@@ -57,9 +57,6 @@ func (r Resolver) BuildID(id string) (b sdktypes.Build, bid sdktypes.BuildID, er
 		return
 	}
 
-	ctx, cancel := limitedContext()
```

Comment:
Are the `limitedContext()` function in line 39, and the `timeout` const in line 18, needed anymore? If not, delete them